Description: workaround for tslib update
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-09-17

--- a/test/ecmascript.ts
+++ b/test/ecmascript.ts
@@ -440,6 +440,7 @@
       throw new Error("should have thrown");
     } catch (e) {
       assert.strictEqual(
+	// @ts-ignore
         e.message,
         'did not recognize object of type "asdf"'
       );
@@ -574,6 +575,7 @@
           this.traverse(path);
         } catch (err) {
           assert.ok(err instanceof this.AbortRequest);
+	  // @ts-ignore
           err.cancel();
         }
       },
@@ -1586,6 +1588,7 @@
 
     } catch (err) {
       assert.strictEqual(
+	// @ts-ignore
         err.message,
         "Must either call this.traverse or return false in visitIdentifier"
       );
--- a/test/typescript.ts
+++ b/test/typescript.ts
@@ -70,10 +70,12 @@
         var options = JSON.parse(fs.readFileSync(
           path.join(path.dirname(fullPath), "options.json")).toString());
       } catch (optionsError) {
+	// @ts-ignore
         console.error(optionsError.message);
       }
 
       if (options &&
+	  // @ts-ignore
           options.throws === error.message) {
         return null;
       }
